mllib/src/main/scala/org/apache/spark/mllib/stat/distribution/MultivariateGaussian.scala

```diff
@@ -15,11 +15,13 @@
  * limitations under the License.
  */
 
-package org.apache.spark.mllib.stat.impl
+package org.apache.spark.mllib.stat.distribution
 
 import breeze.linalg.{DenseVector => DBV, DenseMatrix => DBM, diag, max, eigSym}
 
+import org.apache.spark.mllib.linalg.{Vectors, Vector, Matrices, Matrix}
 import org.apache.spark.mllib.util.MLUtils
+import org.apache.spark.annotation.DeveloperApi;
 
 /**
  * This class provides basic functionality for a Multivariate Gaussian (Normal) Distribution. In
```

Comment:
Please add `:: DeveloperApi ::` before `This class ...`. We need it to generate the doc correctly. You can check other `@DeveloperApi` usage as examples.